meteor/lib/collections/PackageContainerPackageStatus.ts

```diff
@@ -36,11 +35,9 @@ export interface PackageContainerPackageStatusDB {
 	modified: Time
 }
 
-export const PackageContainerPackageStatuses: TransformedCollection<
-	PackageContainerPackageStatusDB,
-	PackageContainerPackageStatusDB
-> = createMongoCollection<PackageContainerPackageStatusDB>('packageContainerPackageStatuses')
-registerCollection('PackageContainerPackageStatuses', PackageContainerPackageStatuses)
+export const PackageContainerPackageStatuses =
+	createMongoCollection<PackageContainerPackageStatusDB, PackageContainerPackageStatusDB>('packageContainerStatuses')
+registerCollection('PackageContainerStatuses', PackageContainerPackageStatuses)
```

Comment:
What is the motivation behind renaming the collection?

Reply:
Agreed, this doesn't seem right. Is there a bad merge somewhere?
`registerCollection('PackageContainerPackageStatuses', PackageContainerPackageStatuses)` is the right one.

Reply:
there was a merge conflict on every one of these lines (formatting vs my change), so yes this will be a bad merge